components/Slider/src/MDCSlider.h

```diff
@@ -214,19 +214,25 @@ IB_DESIGNABLE
 /**
  The number of discrete values that the slider can take.
 
- If greater than or equal to 2, the thumb will snap to the nearest discrete value when the user
- lifts their finger or taps. The discrete values are evenly spaced between the @c minimumValue and
+ The discrete values are evenly spaced between the @c minimumValue and
  @c maximumValue. If 0 or 1, the slider's value will not change when the user releases the thumb.
 
  The default value is zero.
  */
 @property(nonatomic, assign) NSUInteger numberOfDiscreteValues;
 
 /**
- Configures the visibility of the track tick marks.
+ If @c YES and @c numberOfDiscreteValues is greater than 1, the thumb will snap to the nearest
+ discrete value when the user drags the Thumb or taps.
+
+ Defaults to @c YES.
+
+ @note This property has no effect if @c numberOfDiscreteValues is less than 2.
+ */
+@property(nonatomic, assign, getter=isDiscrete) BOOL discrete;
```

Comment:
@wenyuzhang666 This change should be non-breaking, but does change behavior.  I believe that means this particular API would be a "minor" version bump.

Reply:
Thanks for mentioning it. I will do a minor release based on "any public APIs added, then this release is a minor release.". There is also another API added for AppBar.